solana/program_v2/programs/gateway_v2/src/state/network.rs

```diff
@@ -0,0 +1,301 @@
+use crate::errors::NetworkErrors;
+use crate::instructions::*;
+use crate::util::*;
+use anchor_lang::prelude::*;
+use anchor_lang::{AnchorDeserialize, AnchorSerialize};
+use bitflags::bitflags;
+
+/// A gatekeeper network which manages many [`Gatekeeper`]s.
+#[derive(Debug)]
+#[account]
+pub struct GatekeeperNetwork {
+    /// The version of this struct, should be 0 until a new version is released
+    pub version: u8,
+    /// The initial authority key
+    pub initial_authority: Pubkey,
+    // /// Features on the network, index relates to which feature it is. There are 32 bytes of data available for each feature.
+    // pub network_features: Vec<[u8; 32]>,
```

Comment:
Need to implement this? I'd merge anyway and add it in later... maybe a TODO here

Reply:
TODO added